commands/operator-sdk/cmd/build.go

```diff
@@ -173,6 +180,18 @@ func buildFunc(cmd *cobra.Command, args []string) {
 	fmt.Fprintln(os.Stdout, string(o))
 
 	if enableTests {
+		buildTestCmd := exec.Command("go", "test", "-c", "-o", filepath.Join(wd, "tmp/_output/bin", filepath.Base(wd)+"-test"), testLocationBuild+"/...")
+		buildTestCmd.Env = goBuildEnv
+		o, err := buildTestCmd.CombinedOutput()
+		if err != nil {
+			cmdError.ExitWithError(cmdError.ExitError, fmt.Errorf("failed to build: (%v)", string(o)))
+		}
+		fmt.Fprintln(os.Stdout, string(o))
+		// if a user is using an older sdk repo as their library, make sure they have required build files
+		_, err = os.Stat("tmp/build/test-framework/Dockerfile")
+		if err != nil {
```

Comment:
maybe add a logging to indicate that you are missing the` tmp/build/test-framework/Dockerfile` and you are going to `tmp/build/test-framework/Dockerfile`

Reply:
offline discussion: generator already prints out the file it is going to generate. so we don't need to add extra print statement.